Security/Core/Authentication/Token/OAuthToken.php

```diff
@@ -230,7 +230,7 @@ public function isExpired()
             return false;
         }
 
-        return ($this->createdAt + $this->expiresIn - time()) < 30;
+        return (($this->createdAt + $this->expiresIn) - time()) < 30;
```

Comment:
I don't get which is fixed by adding parenthesis on this addition. Could you please elaborate?